src/oss/langchain/multi-agent.mdx

```diff
@@ -213,16 +432,16 @@ const callSubagent1 = tool(
 ```
 :::
 
-### Control the output from the subagent
+</Accordion>
+
+#### Subagent outputs
 
-Two common strategies for shaping what the main agent receives back from a subagent:
+Customize what the main agent receives back so it can make good decisions. Two strategies:
 
-* **Modify the prompt** – Refine the subagent’s prompt to specify exactly what should be returned.
-  * Useful when outputs are incomplete, too verbose, or missing key details.
-  * A common failure mode is that the subagent performs tool calls or reasoning but does **not include the results** in its final message. Remind it that the controller (and user) only see the final output, so all relevant info must be included there.
-* **Custom output formatting** – adjust or enrich the subagent's response in code before handing it back to the main agent.
-  * Example: pass specific state keys back to the main agent in addition to the final text.
-  * This requires wrapping the result in a @[`Command`] (or equivalent structure) so you can merge custom state with the subagent’s response.
+1. **Prompt the sub-agent**: Specify exactly what should be returned. A common failure mode is that the sub-agent performs tool calls or reasoning but doesn't include results in its final message—remind it that the supervisor only sees the final output.
+2. **Format in code**: Adjust or enrich the response before returning it. For example, pass specific state keys back in addition to the final text using a `Command`.
+
+<Accordion title="Subagent outputs">
```

Comment:
```suggestion
<Accordion title="Subagent outputs example">
```

Or could use the `expandable` tag here on the codeblock so it shows some of the code but not all of it? https://www.mintlify.com/docs/create/code#expandable